### DISCUSSION TITLE
ET evaluation problem after the data/dataloader update

### DISCUSSION THREAD

**Discussion by cuteyyt** [FETS-AI/Challenge#125] [Q&A]
Dear Organizers,

After updating the [code](https://github.com/FETS-AI/Challenge/pull/124/files#diffed62650a336fa56e7419966264557266098c71274cd545383672af82fc664285R457) for fixing up the **data/data_loader bug**, we rerun the code with both our custom functions and the notebook's default settings directly and found that **the ET dice score is very low** while the WT and TC seem OK. The figure below shows the last round's evaluation results using our custom functions training 40 rounds,
![image](https://user-images.githubusercontent.com/54923393/132179546-86d3a743-d449-4af8-881d-9bff5be45d5b.png)
which seems abnormal.

Could you kindly help us figure out this issue?

Thanks!

**Comment by hz0567** [comment]
I have the same question. @psfoley @sbakas @sarthakpati @msheller

**Comment by GorkemP** [comment]
We did not encounter such issue for the Task-1 (when running the baseline code).

**Reply by cuteyyt** [reply]
Thanks! We had run more rounds yesterday and the ET scores were up to 60. As we haven't seen this issue before (when the data/data_loader bug hasn't been fixed, we found that our algorithm‘s convergence speed will obviously slow down after 40-60 rounds), so it confused us.

**Reply by GorkemP** [reply]
We face with the high RAM consumption; therefore, get out-of-memory error after certain rounds. Have you experienced such issue when you train with more rounds?

**Reply by cuteyyt** [reply]
We did not encounter the RAM problem. However, we met #106 (program randomly killed) from time to time.
